librad/Cargo.toml

```diff
@@ -50,10 +50,11 @@ url = { version = "2.1", features = ["serde"] }
 urltemplate = "0.1"
 webpki = "0.21"
 yasna = { version = "0.3", features = ["bit-vec"] }
-
+bs58 = "0.3"
+multihash = "0.10"
 
 [dev-dependencies]
 env_logger = "0.7"
 matches = "0.1"
 proptest = "0.9"
-
+futures-await-test = "0.3"
```

Comment:
Can this extra dependency avoided by tapping into the [`tokio::test`](https://docs.rs/tokio/0.2.16/tokio/attr.test.html) macro?

Reply:
As a library, `librad` will continue to pretend that the community will eventually be able to deliver on an executor-agnostic futures abstraction.